jgrapht-core/src/test/java/org/jgrapht/TestEdge.java

```diff
@@ -0,0 +1,94 @@
+/* ==========================================
+ * JGraphT : a free Java graph-theory library
+ * ==========================================
+ *
+ * Project Info:  http://jgrapht.sourceforge.net/
+ * Project Creator:  Barak Naveh (http://sourceforge.net/users/barak_naveh)
+ *
+ * (C) Copyright 2003-2009, by Barak Naveh and Contributors.
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+/* -------------------------
+ * TestEdge.java
+ * -------------------------
+ * (C) Copyright 2003-2016, by Christoph Zauner and Contributors
+ *
+ * Original Author:  Christoph Zauner
+ *
+ * $Id$
+ *
+ * Changes
+ * -------
+ *
+ */
+package org.jgrapht;
```

Comment:
I guess this should live in org.jgrapht.graph instead (since that's where the DefaultEdge class is defined).